charts/hush-sensor/templates/sensorconfigmap.yaml

```diff
@@ -25,3 +25,16 @@ data:
   org_id: {{ $di.orgId | quote }}
   deployment_id: {{ $di.deploymentId | quote }}
   event_reporting_console: {{ .Values.eventReportingConsole | quote }}
+  crawler_config: |
+    {{- if and .Values.sentry.crawler .Values.sentry.crawler.aws }}
+    {{- $awsMode := .Values.sentry.crawler.aws.mode }}
+    aws:
+      {{- if eq $awsMode "irsa" }}
+      mode: "default"
+      {{- else if eq $awsMode "assume_role" }}
+      mode: "assume_role"
+      iam_role: {{ .Values.sentry.crawler.aws.iam_role }}
+      {{- else }}
```

Comment:
IMO we shouldn't include any reference to `assume_role` mode as long as it is not supported in the service (in code and in commit message). Will it be supported by the time this chart is released?